### ISSUE TITLE
Блок с ползунком «Сделать доступным из любой точки сценария» может работать некорректно.

### ISSUE THREAD

**Issue by Sergeycul** [sberdevices/salute-issues#83]
ГРАФ. Проблема с блоком «Помощь». Т.к. этот блок должен отрабатывать по ходу всего апа, то в конструкторе ставим ползунок «Сделать доступным из любой точки сценария». НО! В тогда самом блоке нельзя использовать интент «Любая другая фраза», т.к. этот интент может перебивать по ходу апа другие блоки с интентами, где тоже используется «Любая другая фраза». Желательно сделать так, чтобы при ползунке «Сделать доступным из любой точки сценария» в этом блоке интент «Любая другая фраза» отрабатывал только в этом блоке, а не по всему сценарию.

**Comment by asmalakhov-sberdevices** [comment]
@Sergeycul я же правильно понимаю, речь о блоке Интенты, и о том, что не срабатывает Любая фраза - если блок глобальный?

**Comment by Genenko-Svetlana** [comment]
@Sergeycul

**Comment by Sergeycul** [comment]
Нет, фраза как раз срабатывает и конкурирует с другими интентами "Любая другая фраза" в других блоках. Желательно, чтобы тут эта фраза была только для этого блока и не встраивалась по всему сценарию.

**Comment by Genenko-Svetlana** [comment]
@asmalakhov-sberdevices

**Comment by asmalakhov-sberdevices** [comment]
@Sergeycul в данный момент, любая фраза в глобальных блоках по факту не работает. И срабатывает интент, так как однозначно нельзя в момент определить, в каком именно глобальном блоке она должно сработать. 
Мы планировали скрывать переход Любая другая фраза для глобальных блоков, чтобы она не путала при разработке. 

Переход любая другая фраза актуален только не для глобальных интентов. 

Мы взяли задачи в анализ, возможно сделаем лучшее решение, чем просто скрыть переход для Глобальных блоков.

**Comment by Genenko-Svetlana** [comment]
Срок 31.03.2021

**Comment by Genenko-Svetlana** [comment]
Возвращаем в работу.
